k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/SelectOrExamineResponseTest.java

```diff
@@ -141,20 +144,15 @@ public void newInstance_withoutOkResponse_shouldReturnNull() throws Exception {
         assertNull(result);
     }
 
-    private static List<ImapResponse> createMockImapResponses(String lastLine) throws IOException {
-        return asList(
-                createImapResponse("* 23 EXISTS"),
-                createImapResponse("* 0 RECENT"),
-                createImapResponse("* OK [UIDVALIDITY " + TEST_UIDVALIDITY + "] UIDs valid"),
-                createImapResponse("* OK [UIDNEXT 57576] Predicted next UID"),
-                createImapResponse("* OK [HIGHESTMODSEQ " + TEST_HIGHESTMODSEQ + "] Highest mailbox mod-sequence"),
-                createImapResponse("* FLAGS (\\Answered \\Flagged \\Deleted \\Seen \\Draft NonJunk $MDNSent)"),
-                createImapResponse("* OK [PERMANENTFLAGS (\\Answered \\Flagged \\Deleted \\Seen \\Draft NonJunk " +
-                        "$MDNSent \\*)] Flags permitted."),
-                createImapResponse("* VANISHED (EARLIER) 41,200,230:233"),
-                createImapResponse("* 49 FETCH (UID 117 FLAGS (\\Seen \\Answered) MODSEQ (90060115194045001))"),
-                createImapResponse("* 50 FETCH (UID 119 FLAGS (\\Draft $MDNSent) MODSEQ (90060115194045308))"),
-                createImapResponse(lastLine)
-        );
+    private List<ImapResponse> createMockImapResponses(String lastLine) throws IOException {
```

Comment:
related to createFolderOpenedResponse comment above: if you compare this new code to the old one, it's very hard to tell what is going on exactly. in the old code, it's just a plain response that is simple to parse.

the `createMockImapResponses` method serves a similar purpose to the `createFolderOpenedResponse` one, but is much less complex.